### PR TITLE
Add test licence declaration

### PR DESCRIPTION
**Pull request by dougajmcdonald**
Also resolve bug with trading names showing in the summary when `HasTradingName` was null/false

### CODE REVIEW

File: GLAA.Web/Views/Shared/DisplayTemplates/OrganisationDetailsViewModel.cshtml [hunk 1]
```diff
@@ -19,22 +19,44 @@
...
             <td>@Html.LabelFor(m => m.BusinessName.TradingName)</td>
-            <td class="@Model.BusinessName?.TradingName">@Html.DisplayFor(m => m.BusinessName.TradingName, new { @class = "org" })</td>
+            <td class="@Model.BusinessName?.TradingName">
+            @if (Model.BusinessName.HasTradingName ?? false)
+            {
+                @Html.DisplayFor(m => m.BusinessName.TradingName, new { @class = "org" })
+            }
+            else
+            {
+                var text = "Not set";
```
Comment (SeiyaYamamotoStudios) [inline GLAA.Web/Views/Shared/DisplayTemplates/OrganisationDetailsViewModel.cshtml]: Do we need this variable?

**Comment by SeiyaYamamotoStudios** [comment]
perhaps its my misinterpretation with the variable - but other than that it looks good.

File: GLAA.Web/Views/Shared/DisplayTemplates/OrganisationDetailsViewModel.cshtml (hunk 1 above)
Comment (dougajmcdonald) [reply GLAA.Web/Views/Shared/DisplayTemplates/OrganisationDetailsViewModel.cshtml]: I think we do unless it's html wrapped, now I think about it I could have done that in the first place! missed it!

Comment (SeiyaYamamotoStudios) [reply GLAA.Web/Views/Shared/DisplayTemplates/OrganisationDetailsViewModel.cshtml]: Oh yeah, I see what you mean.

**Review by SeiyaYamamotoStudios — APPROVED** [review]

**Pull request merged**